src/collective/instancebehavior/tests/test_instancebehavior.py

```diff
@@ -1,4 +1,5 @@
-import unittest2 as unittest
+# -*- coding: utf-8 -*-
```

Comment:
@rnixx fair point, the question is if people will remember to add it as soon as they add a non-ASCII character... so mostly a safety belt